assets/js/components/Work/Tabs/Administrative.jsx

```diff
@@ -143,23 +147,33 @@ const WorkTabsAdministrative = ({ work }) => {
   }
 
   return (
-    <form name="work-administrative-form" onSubmit={handleSubmit(onSubmit)}>
+    <form
+      name="work-administrative-form"
+      data-testid="work-administrative-form"
+      onSubmit={handleSubmit(onSubmit)}
+    >
       <UITabsStickyHeader title="Administrative Metadata">
         {!isEditing && (
           <button
             type="button"
             className="button is-primary"
             onClick={() => setIsEditing(true)}
+            data-testid="edit-button"
           >
             Edit
           </button>
         )}
         {isEditing && (
           <>
-            <button type="submit" className="button is-primary">
+            <button
```

Comment:
Quick thing...can you pull in the new `<Button />` component from the our Admin UI Components repo?   Just to get us in the habit of starting to use that component repo and minimize our surface area for possible bugs.

See an example here:  https://github.com/nulib/meadow/blob/194df35a2bf56d6f57198cff2d2530085f9db5aa/assets/js/components/Project/Form.jsx#L10